testgen/src/header.rs

```diff
@@ -36,6 +36,9 @@ pub struct Header {
     pub proposer: Option<usize>,
     #[options(help = "last block id hash (default: Hash::None)")]
     pub last_block_id_hash: Option<Hash>,
+    #[options(help = "application hash (default: AppHash(vec![])")]
+    #[serde(default, with = "app_hash_serde")]
+    pub app_hash: Option<AppHash>,
```

Comment:
What's the intent here? If the `app_hash` field is `None`, would you rather omit the field from JSON, or serialize it as `null`?
The former you can get with `#[serde(skip_serializing_if="Option::is_none")]`.

Reply:
Skipped if none: [73e8e14](https://github.com/informalsystems/tendermint-rs/pull/1344/commits/73e8e14e74cd84bb9c423930b0382c71d7dd7f7c)